app/views/reviewer_mailer/assignment.html.haml

```diff
@@ -15,7 +15,7 @@
   The deadline for your review is #{@grant.review_close_date.strftime("%B %e, %Y")}.
 
 %p
-  By clicking the link above you can review or opt out of reviewing.
+  Review or opt out of reviewing here: #{link_to @submission.title, grant_submission_url(@grant, @submission)}.
```

Comment:
Since the link is above, consider changing this test to something like:
"You may also opt-out of reviewing this submission at the link above."